ote/src/cljs/ote/views/transit_visualization/change_icons.cljs

```diff
@@ -39,23 +40,33 @@
 
 
 ;; Ui
+(defn labeled-icon
```

Comment:
Apparently there was a problem with ´ote.ui.icon_labeled´ and it couldn't be used or developed further?

Reply:
ah. My bad. This "labeled-icon" was a copy&paste of ote.ui.icon_labeled "icon-labeled" function.
And it was used in transit_visualization view. So I only moved it to change_icon file.

So I now removed that copy&paste and take icon-labeled into use.